modules/richaudienceBidAdapter.js

```diff
@@ -138,3 +127,42 @@ export const spec = {
 };
 
 registerBidder(spec);
+
+function raiGetSizes(bid) {
+  let raiNewSizes;
+  if (bid.mediaTypes && bid.mediaTypes.banner && bid.mediaTypes.banner.sizes) {
+    raiNewSizes = bid.mediaTypes.banner.sizes
+  } else {
+    raiNewSizes = bid.sizes
+  }
+  if (raiNewSizes != null) {
+    return raiNewSizes.map(size => ({
+      w: size[0],
+      h: size[1]
+    }));
+  }
+}
+
+function raiSetEids(bid) {
+  let eids = [];
+
+  if (bid && bid.userId) {
+    raiSetUserId(bid, eids, 'id5-sync.com', utils.deepAccess(bid, `userId.id5id`));
```

Comment:
in this method call, you're passing in 4 arguments (`bid`, `eids`, `source`, and `value`), however the method definition on line 161 also has a 5th argument for `userId`. I'm guessing the definition doesn't need the last argument, but wanted to point it out so you can fix.